x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -437,12 +436,12 @@ export class HeadlessChromiumDriver {
     // In order to get the inspector running, we have to know the page's internal ID (again, private)
     // in order to construct the final debugging URL.
```

Comment:
I think this whole method is no longer working: https://github.com/elastic/kibana/issues/101233

we should just remove it soon

Reply:
Nice, all those private underscore accesses don't seem like they will work for long.